### PR TITLE
Gallery

### PR DESCRIPTION
**Pull request by j-vent**
Added new page for photos.

### CODE REVIEW

File: portfolio/src/main/webapp/cook.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <title>Food Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+ <body>
+
+ <div class="foodcontent" >
+     <br/>
+    <h2 class="center">Food Gallery</h2>
+    <img class="pfp center" style="width:250px;height:250px;" src="img/shef.png" />
```
Comment (landondare) [inline portfolio/src/main/webapp/cook.html:14]: Generally, inline styling is not recommended. Is there a reason you can't include this info in the "pfp center" class?

File: portfolio/src/main/webapp/cook.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <h3 class="center">Some food I've made over quarantine ... </h3>
+
+    <div id="imggrid">
+        <img class="foodpic" src="img/rice.jpg"/>
```
Comment (landondare) [inline portfolio/src/main/webapp/cook.html:18]: Consider trying to automate some of these repeated  tags with some javascript. You could create a list with all the image srcs and then append them programmatically to the imiggrid div

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -38,17 +37,17 @@ <h2 class="center">About</h2>
           Currently I am a STEP intern at Google, and I am excited to learn more about Javascript and web dev in general. 
           Other areas of interest are machine learning and computer vision. 
       </p>
-      <p >In my free time I like to relax by playing acoustic guitar and playing Overcooked! and CS:GO with my friends. 
+       <p >In my free time I like to relax by playing acoustic guitar and playing Overcooked! and CS:GO with my friends.
```
Comment (landondare) [inline portfolio/src/main/webapp/index.html:40]: \<p\ > --> \<p\> (remove the space)

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -38,17 +37,17 @@ <h2 class="center">About</h2>
...
-      </p>
-      <div class="blogpost">
+          trivia (I played on my high school's team and I'd love to be on Jeopardy! someday), makeup, and <a href="/cook.html">cooking</a>.  
+        </p>
+        <div class="blogpost">
         <p>I don't have any pets of my own, though I do really like bunnies. 
             Here's a photo of a cute rabbit that you can pet (by clicking):</p>
          <img id="bunny" onclick="bunnymsg()" style="height: 50px;width:80px;" src="img/bunny.jpg" alt="white rabbit"/>
-        <div id="bun_msg" style=" color:darkslategrey;"></div>
+         <div id="bun_msg" style=" color:darkslategrey;"></div>
```
Comment (landondare) [inline portfolio/src/main/webapp/index.html:50]: Your indentation looks a bit weird on this line and some of the lines above it. Try to maintain a 2 space independent between each container. e.g.
```
<div1>
  <div2>
    <div3> ....
```

File: portfolio/src/main/webapp/cook.html (hunk 1 above)
Comment (j-vent) [reply portfolio/src/main/webapp/cook.html:14]: Yeah I tried to remove most inline styling but I thought I could get away with it for sizing,  but I changed this and others I found.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (j-vent) [reply portfolio/src/main/webapp/index.html:40]: whoops didn't notice this

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (j-vent) [reply portfolio/src/main/webapp/index.html:50]: Yes, I missed this section when I went to fix my indents thanks for pointing out.

File: portfolio/src/main/webapp/cook.html (hunk 2 above)
Comment (j-vent) [reply portfolio/src/main/webapp/cook.html:18]: I tried automating it w/ JS but the imggrid layout is not retained. This is probably best for a straight layout.

Comment (landondare) [reply portfolio/src/main/webapp/cook.html:18]: Thats ok, you would probably have more luck if we were using a frontend framework anyway. This is fine for now.

**Comment by landondare** [comment]
LGTM, will wait for Zach to review and approve

**Pull request merged**